python/numpy/timings/black_scholes_weld.py

```diff
@@ -0,0 +1,111 @@
+# Implements the Black Scholes pricing model using NVL.
+#
+# The following links were used to define the constants c05 and c10:
+#
+# http://codereview.stackexchange.com/questions/108533/fastest-possible-cython-for-black-scholes-algorithm
+# http://gosmej1977.blogspot.com/2013/02/black-and-scholes-formula.html
+
+import numpy as np
+# import weld_numpy_elemwise as nnp
+import sys
+sys.path.append('..')
+from weldarray import WeldArray
+import os
+import time
+import random
+
+invsqrt2 = 1.0 #0.707
+verbose = False
+
+os.environ["NUM_WORKERS"] = "12"
+
+import black_scholes_config as bsc
+
+def black_scholes_nvl(price, strike, t, rate, vol):
```

Comment:
that had just been left over from the old version, I'll make the change to the names in this file